fe/fe-core/src/main/java/org/apache/doris/catalog/InternalSchemaInitializer.java

```diff
@@ -154,11 +153,11 @@ public CreateTableStmt buildHistogramTblStmt() throws UserException {
         columnDefs.add(new ColumnDef("buckets", TypeDef.createVarchar(ScalarType.MAX_VARCHAR_LENGTH)));
         columnDefs.add(new ColumnDef("update_time", TypeDef.create(PrimitiveType.DATETIME)));
         String engineName = "olap";
-        KeysDesc keysDesc = new KeysDesc(KeysType.UNIQUE_KEYS,
-                Lists.newArrayList("id"));
+        ArrayList<String> uniqueKeys = Lists.newArrayList("id", "catalog_id",
```

Comment:
Why changed this?

Reply:
TThis is to use different dimensions to delete statistics, otherwise the UNIQUE mode can only be deleted by id.